components/TheHeader.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+    <div class="header">
+        <div >
+            <svg xmlns="http://www.w3.org/2000/svg" width="34" height="34" viewBox="0 0 34 34" fill="none">
+                <g clip-path="url(#clip0_8_429)">
+                <path d="M33.5793 4.4657C29.9019 1.67903 24.2431 0.0153503 18.4416 0.0153503C11.2658 0.0153503 5.4185 2.52072 2.39921 6.88906C0.981168 8.94068 0.196844 11.3701 0.0680823 14.1099C-0.0466013 16.5494 0.363059 19.2473 1.28611 22.1499C4.43622 12.7074 13.2333 5.31251 23.375 5.31251C23.375 5.31251 13.8847 7.81045 7.91816 15.5454C7.91444 15.55 7.83535 15.648 7.69856 15.8342C6.50059 17.4372 5.45609 19.2595 4.67448 21.3237C3.35068 24.4723 2.12502 28.7926 2.12502 34H6.37501C6.37501 34 5.72988 29.9419 6.85181 25.2746C8.70753 25.5254 10.3666 25.65 11.8606 25.65C15.7678 25.65 18.5467 24.8046 20.6059 22.9896C22.4509 21.3634 23.4679 19.1783 24.5446 16.8651C26.1889 13.332 28.0526 9.32763 33.4647 6.23502C33.7745 6.05799 33.9748 5.73724 33.9978 5.38117C34.0209 5.0251 33.8637 4.68125 33.5793 4.4657Z" fill="#7A8DFD"/>
+                </g>
+                <defs>
+                <clipPath id="clip0_8_429">
+                <rect width="34" height="34" fill="white"/>
+                </clipPath>
+                </defs>
+            </svg>
+            <span class="headerText">Todos</span>
+        </div>
+        <div style="display:flex;">
+            <div @click="startAnim" id="searchIcon">
+                <svg  xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+                    <path d="M23.111 20.058L18.134 15.081C19.099 13.561 19.657 11.759 19.657 9.83C19.657 4.41 15.248 0 9.828 0C4.408 0 0 4.41 0 9.83C0 15.25 4.408 19.66 9.829 19.66C11.663 19.66 13.381 19.155 14.851 18.277L19.872 23.298C22.016 25.439 25.256 22.202 23.111 20.058ZM3.047 9.83C3.047 6.091 6.09 3.048 9.829 3.048C13.568 3.048 16.611 6.09 16.611 9.83C16.611 13.57 13.568 16.612 9.829 16.612C6.09 16.612 3.047 13.569 3.047 9.83ZM5.057 8.066C7.041 3.467 13.721 4 14.979 8.815C12.445 5.841 7.986 5.521 5.057 8.066Z" fill="#BBBDD0"/>
+                </svg>
+                <input v-if="showInput" v-model="searchInput" class="inputField" placeholder="enter todo name" @change="handleChange" />
+            </div>
+        <select class="" @change="localePathAbout">
+            <option v-for="locale in $i18n.locales" :key="locale.code">
+                <!-- <nuxt-link :to="localePathAbout(locale.code)"> -->
+                    {{locale.code}}
+                <!-- </nuxt-link> -->
+            </option> 
+        </select>
+        </div>
+    </div>
+</template>
+<script>
+export default {
+    name:'TheHeader',
+    data() {
+        return {
+            searchInput: '',
+            showInput: false,
+            setTime: ''
+        }
+    },
+    methods: {
+    localePathAbout(event) {
+       if(event.target.value === 'en')
+      {return  this.$router.replace(this.switchLocalePath('en'))}
```

Comment:
Can we skip route paths when translating?

Reply:
yes. Resolved